src/BenchmarksApps/TodosApi/TodosApi.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <ServerGarbageCollection>false</ServerGarbageCollection>
+    <InvariantGlobalization>true</InvariantGlobalization>
+    <UserSecretsId>b8ffb8d3-b768-460b-ac1f-ef267c954c85</UserSecretsId>
+    <PublishAot>true</PublishAot>
```

Comment:
In the BasicMinimalApi app we pass this during the build. I kind of like having it in the .csproj though, for ease of development (it's what we are trying to achieve).

Do we plan on trying non-AOT scenarios in the benchmarks?

Reply:
I added it for the same reason you mention, i.e. having it on during development. We'll still be measuring non-AOT in benchmarks to calculate the delta between CoreCLR vs. native AOT.

Reply:
I set this back to `false` for now as the data access doesn't work due to the JSON issue.